src/program/example/example.lua

```diff
@@ -0,0 +1,17 @@
+module(..., package.seeall)
+
+local ffi = require("ffi")
+local C = ffi.C
+local lib = require("core.lib")
+local usage = require("program.example.README_inc")
+
+function run (args)
```

Comment:
Agree, I just blindly copied the `snabbnfv` hierarchy without thinking.

> Stop using subcommands and simply flatten out the command-name space. (s/snabbnfv traffic/snabbnfv-traffic/)

I think this is the smartest move, we're not exactly running out of namespace are we?